src/Workspaces/Core/Desktop/Workspace/MSBuild/ProjectFile/ProjectAnalyzerReference.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.MSBuild
+{
+    /// <summary>
+    /// Represents a reference to another project file.
```

Comment:
This the analyzer reference info coming out of the project file, not to be confused with an actual AnalyzerReference type.  It does need to be serializable, but it also needs to not be constructed yet, which happens later with the command line compiler.